now.js

```diff
@@ -1,13 +1,4 @@
-// Based on https://github.com/chadfawcett/probot-serverless-now/blob/master/index.js
-const { createProbot } = require('probot')
-const { getPrivateKey } = require('@probot/get-private-key')
+const { createNodeMiddleware, createProbot } = require('probot')
 
 const app = require('./')
-const probot = createProbot({
-  id: process.env.APP_ID,
-  secret: process.env.WEBHOOK_SECRET,
-  cert: getPrivateKey()
-})
-probot.load(app)
-
-module.exports = probot.server
+module.exports = createNodeMiddleware(app, createProbot())
```

Comment:
Changes based on https://probot.github.io/docs/serverless-deployment/#deploy-the-app.